test/externalTests/bleeps.sh

```diff
@@ -37,7 +37,7 @@ function test_fn { HARDHAT_DEPLOY_FIXTURE=true npx --no hardhat --no-compile tes
 
 function bleeps_test
 {
-    local repo="https://github.com/wighawag/bleeps"
+    local repo="https://github.com/wighawag/bleeps.git"
```

Comment:
Does it make any difference? The link without it looks cleaner so I'd leave it as is if it works.

Reply:
No, I just wanted to follow the same standard that github uses. But I will remove it, no problem.